cashaddr.md

```diff
@@ -119,6 +119,19 @@ Allowing for uppercase ensure that the address can be encoded efficiently in QR
 
 In some contexts, such as payment URLs or QR codes, the addresses are currently prefixed with `bitcoincash:`. In these contexts, the address must not be double prefixed.
 
+## Examples of address transations
+
+The following addresses are given into the legacy and new format.
```

Comment:
+The following addresses are given in the legacy and corresponding CashAddr format.